keep/common/fedora.py

```diff
@@ -319,8 +319,12 @@ def save(self, logMessage=None):
 
                 raise DuplicateContent(msg, pids, pid_cmodels)
 
-        return super(DigitalObject, self).save(logMessage)
+        # check if the mods content title of the object has been changed
+        # if changed, apply the change to the object in pidman
+        if self.mods.isModified(): # if title is changed it'd be reflected in mods
+            pidman.update_ark(noid=self.noid, name=self.mods.content.title)
```

Comment:
I like how you had this pulled out into a method before, I think that is still useful.  Could be used in the manage script, and would allow us to override if we identify any content models that don't use mods for the title.

Also, I _think_ it might be worth checking the pidman title first and seeing if it needs changed (possibly I told you the opposite before); I'm concerned that updating the pid with no real change will result in a confusing history trail for the pid in pidman.  This is something we could probably check fairly easily by testing a couple of objects (or it may get uncovered in acceptance testing on the update on edit).